src/storage/txn/process.rs

```diff
@@ -824,6 +823,31 @@ fn process_write_impl<S: Snapshot>(
             statistics.add(&txn.take_statistics());
             (ProcessResult::Res, txn.into_modifies(), rows, ctx, None)
         }
+        Command::RefreshLock {
+            ctx,
+            key,
+            start_ts,
+            options,
+        } => {
+            // start_ts + 1 because we want to read a lock with ts = start_ts
+            let mut txn = MvccTxn::new(snapshot.clone(), start_ts + 1, !ctx.get_not_fill_cache())?;
```

Comment:
Why + 1?, and `snapshot` need not be cloned.

Reply:
Now when it receives such a request with `start_ts = T`, it actually finds the lock whose `start_ts = T + 1`. I guess this is not what we need.